app/src/main/res/layout/activity_manifest_viewer.xml

```diff
@@ -1,13 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
     tools:context=".manifest.ManifestViewerActivity"
     tools:ignore="ContentDescription">
 
-    <include layout="@layout/toolbar" />
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <include layout="@layout/toolbar" />
+
+        <LinearLayout
```

Comment:
When the search is activated the view jumps because the height is 1 dp bigger than the toolbar height.